.travis.yml

```diff
@@ -20,6 +20,7 @@ before_install:
   - ulimit -a
   - python -V
   - pip install -q numpy psutil
+  - pip install -r requirements-rtfd.txt
```

Comment:
The name of the file is `requirements_rtfd.txt`, not `requirements-rtfd.txt`.

Reply:
Replaced with `requirements-dev.txt`